scripts/resultsSum.pl

```diff
@@ -160,28 +160,38 @@ sub resultReporter {
 						$tapString .= "not ok " . $numOfTotal . " - " . $testName . "\n";
 						$tapString .= "  ---\n";
 						if (($diagnostic eq 'failure') || ($diagnostic eq 'all')) {
-							if ($buildList =~ /openjdk/) {
+							if ($diagnostic eq 'all') {
+								$tapString .= $output;
+							}
```

Comment:
Looks it is missing one case: if diagnostic eq 'failure' and buildList is not openjdk or jck, we still need to append output to tapString.